docs/content/en/docs/getting-started/_index.md

```diff
@@ -54,4 +54,33 @@ helm install k8ssandra-tools k8ssandra/k8ssandra
 helm install k8ssandra-cluster-a k8ssandra/k8ssandra-cluster  
 ```
 
+## Upgrade K8ssandra
 In later steps, you can upgrade your k8ssandra-cluster via `helm upgrade` commands, for example to access services from outside Kubernetes via a Traefik Ingress controller.
+
+```console
+# Upgrade shared dependencies / tooling
+helm upgrade k8ssandra-tools k8ssandra/k8ssandra
+
+# Upgrade the K8ssandra cluster created before: "k8ssandra-cluster-a" 
+helm upgrade k8ssandra-cluster-a k8ssandra/k8ssandra-cluster  
+```
+
+
+## Uninstall K8ssandra
+To uninstall the K8ssandra, just need to run below:
+
+```console
+# Uninstall shared dependencies / tooling
+helm uninstall k8ssandra-tools k8ssandra/k8ssandra
+
+# Uninstall the K8ssandra cluster named "k8ssandra-cluster-a" 
+helm uninstall k8ssandra-cluster-a k8ssandra/k8ssandra-cluster  
+```
+
+Delete the below CRDs manually:
```

Comment:
It may be worth noting that Helm does not delete CRDs. And we should probably list all of the CRDs installed by k8ssandra. There are others from prometheus-operator and from grafana-operator.

Reply:
I've hit this issue as well. :/